src/thermo/Phase.cpp

```diff
@@ -843,6 +843,35 @@ void Phase::modifySpecies(size_t k, shared_ptr<Species> spec)
     invalidateCache();
 }
 
+bool Phase::addSpeciesAlias(const std::string& name, const std::string& alias)
+{
+    size_t k = speciesIndex(name);
+    if (k != npos) {
+        m_speciesIndices[alias] = k;
```

Comment:
One issue with this implementation that I noticed is that you can overwrite the entry that corresponds to a species, leaving it without a way to be referenced by name.

Reply:
This is indeed the case. I added a check to make sure that `alias` does not already exist.